appgate/types.py

```diff
@@ -299,3 +386,31 @@ class MissingFieldDependencies:
     parent_type: str = attrib()
     field_path: str = attrib()
     dependencies: FrozenSet[str] = attrib(factory=frozenset)
+
+
+def get_tags(tags: List[str], tags_env: str) -> Optional[FrozenSet[str]]:
+    xs = frozenset(tags) if tags else frozenset()
```

Comment:
tags is always a list so why not write just:
```suggestion
    xs = frozenset(tags)
```